iris/tests/download_manager/clear_download_history.py

```diff
@@ -64,6 +76,5 @@ def run(self):
 
     def teardown(self):
         # Cleanup.
-        cleanup_list = ['5MB.zip', '10MB.zip']
-        for i in range(len(cleanup_list)):
-            downloads_cleanup(cleanup_list[i])
+        path = IrisCore.get_downloads_dir()
+        downloads_cleanup(path)
```

Comment:
download_cleanup() function shouldn't have a path parameter. The name is self-explanatory that it refers to the downloads directory